src/store/exolix/exolix-epics.ts

```diff
@@ -0,0 +1,60 @@
+import { combineEpics, Epic } from 'redux-observable';
+import { from, Observable, of } from 'rxjs';
+import { catchError, concatMap, map, switchMap } from 'rxjs/operators';
+import { Action } from 'ts-action';
+import { ofType, toPayload } from 'ts-action-operators';
+
+import {
+  // loadExolixCurrencies$,
```

Comment:
remove comment

Reply:
look below for another setTimeout calling after execution current timer: timeoutId = setTimeout(repeat, 3000);